components/automate-deployment/pkg/server/server.go

```diff
@@ -423,9 +423,8 @@ func (s *server) configRenderer() (ConfigRenderer, error) {
 
 		preparableCfg, found := s.deployment.Config.PlatformServiceConfigForService(service.Name())
 		if !found {
-			err := errors.Errorf("unable to render configuration for unknown service %q", service.Name())
-			logrus.WithField("service", service.Name).WithError(err).Error("unable to render configuration")
-			return "", err
+			logrus.WithField("service", service.Name).WithError(err).Warnf("unable to render configuration for unknown service %q", service.Name())
```

Comment:
Why are we OK with this just being a warning?

While we are touching this, `service.Name` here should probably be `service.Name()` (or we could just remove it since we also include the service name in the log message.

Reply:
it was annoying having to define all the protobuf config stuff while i was wiring stuff up. i just wanted it to use the default.toml and start the hab package. I can change it back if it is too scary